tests/core/channels/test_facebook.py

```diff
@@ -0,0 +1,119 @@
+import logging, pytest
+from rasa.core.channels.facebook import MessengerBot
+from fbmessenger import MessengerClient
+
+from _pytest.monkeypatch import MonkeyPatch
+
+import rasa.core.run
+from rasa.core import utils
+
+
+# this is needed so that the tests included as code examples look better
+from tests.utilities import json_of_latest_request, latest_request
+
+logger = logging.getLogger(__name__)
+
+
+def test_facebook_channel():
```

Comment:
Because this test was lifted from `tests/core/test_channels.py` could you please remove it from that file so that there are no duplicates?